store/demo/schema-migration-history/10001__data.sql

```diff
@@ -0,0 +1,196 @@
+-- schema migration seeding data
+INSERT INTO "public"."principal" ("id", "row_status", "creator_id", "created_ts", "updater_id", "updated_ts", "type", "name", "email", "password_hash") VALUES
```

Comment:
This file is exported by TablePlus. It has some different formats than our seed files. But it works well and looks fine to me, so should I need to do format it at all? @d-bytebase

Reply:
Let's define a way to export the demo data and keep the format consistent. But let's worry about it later.